var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -100,6 +107,14 @@ class Libfabric(AutotoolsPackage):
     depends_on("libtool", when="@develop", type="build")
 
     conflicts("@1.9.0", when="platform=darwin", msg="This distribution is missing critical files")
+    conflicts("fabrics=bgq", when="@:1.14.99")
+    conflicts("fabrics=net", when="@:1.14.99")
+    conflicts("fabrics=opx", when="@:1.14.99")
+    conflicts("fabrics=perf", when="@:1.14.99")
+    conflicts("fabrics=rstream", when="@:1.14.99")
```

Comment:
These two and BGQ have been in the code for a while now.

Reply:
I did not check for the exact version support, and only went back to v1.15.0. If those fabrics were left out because of build problems, then then they should be removed, otherwise one could determine the exact minimum version. I'm no expert, however -- all I want is a more recent version of libfabric.

Reply:
They aren't broken but I don't know the semver->provider mapping off the top of my head either, would have to parse through the git log. Why don't we just get the new release in with this PR and bring in the additions separately?

Reply:
OK, I'll fix up this PR to just add the new versions.